content/trust-management/trust-authority.md

```diff
@@ -60,23 +56,163 @@ its list of public signature verification keys by removing the compromised key
+        "@badsource:example.com": {
+            "reason_code": "B_MISINFORMATION",
+            "since": "",
+            "additional_info": {
+                "en": "This source actually belongs to the Tomainian government which uses it for propaganda.",
+                "fr": "Cette source appartient au gouvernement Tomanien qui l'utilise à des fins de propagande."
+            }
+        }
+    }
+}
+```
+
+## Blacklisting
+
+A trust authority **can** list sources and trust authorities that it explicitly
+blacklists for being compromised or ethical reasons. The trust authority
+**must** specify a reason for blacklisting a source or TA, which consists in a
+defined reason code that client implementations **can** use in order to warn
+users about a specific source. The trust authority **can** also provide
+additional information to explain the addition to the blacklist, which, if
+provided, **must** be contained in a custom localised string. Client
+implementations **can** also use this string to provide the users with more
+information on why a specific source was blacklisted by the trust authority.
+
+### Blacklist reason codes
+
+As defined above, a trust authority **must** provide a reason for blacklisting a
+source or another trust authority. This **must** be done using at least a reason
+code, which **must** be one of the following:
+
+|        Code        |            Meaning             |
+| ------------------ | ------------------------------ |
+| `B_COMPROMISED`    | One of the public keys of the source or trust authority has been compromised. |
+| `B_MISINFORMATION` | If the blacklisted entity is a source, it has been publishing false or unverified information intentionnally. If it's a trust authority, it has been certifying sources publishing false or unverified information as trustworthy intentionnally. |
+| `B_ABANDONED`      | The source or trust authority has ceased its activity and/or to publish articles through this federation, therefore the entity isn't used anymore and very unlikely to ever be used again. Blacklisting it then prevents someone else from getting their hands on the entity's keys and tokens and impersonate its former owner. |
+
+## Suggested trust authorities
 
 In order to guide a new user through building his trusted network when they
 enters a Matrix room (i.e. an Informo federation), the room's administrator
-**can** provide a list of suggested TAs (2️⃣: stored in Matrix room state).
-These TAs will be proposed to the users if they doesn't know which TA they
-should trust first.
-
-As an additional security step, the client implementation maintainer **should**
+**can** provide a list of suggested TAs through the publication of a
+`network.informo.suggested_trust_authorities` state event. The event's state key
+**must** be an empty string. The Matrix room's [power
+levels](https://matrix.org/docs/spec/client_server/r0.4.0.html#m-room-power-levels)
+**must** be set so only its administrator can emit a
```

Comment:
In the event of a community managed room, administrator(s) could require PL 50 in order to allow other trusted people to contribute to the list.

I'd write something like this: `The Matrix room's [power levels](https://matrix.org/docs/spec/client_server/r0.4.0.html#m-room-power-levels) **must** be set to deny modifications of this event from unauthorized users (i.e. requiring a power level strictly higher than 0)`